src/application/user/controller.go

```
@@ -64,14 +66,54 @@ func (ctrl *controller) CreateUser(c *gin.Context) {
+	}
+
+	var dto UpdateUserDto
+	if err := c.ShouldBindJSON(&dto); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"error": "invalid request",
+		})
+
+		return
+	}
+	user, gerr := ctrl.service.UpdateUser(id, dto)
+	if gerr != nil {
```

golangci-lint (1.64.8)

only one cuddle assignment allowed before if statement

(wsl)